biojava-genome/src/main/java/org/biojava/nbio/genome/util/ChromosomeMappingTools.java

```diff
@@ -0,0 +1,1394 @@
+package org.biojava.nbio.genome.util;
+
+
+import com.google.common.collect.Range;
+
+import org.biojava.nbio.core.sequence.ProteinSequence;
+import org.biojava.nbio.genome.parsers.genename.ChromPos;
+import org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition;
+
+
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *  A class that take care of the painful mapping
+ */
+public class ChromosomeMappingTools {
+
+    public static final boolean debug = false;
+    private static final String newline = System.getProperty("line.separator");
+
+
+    public static final String CHROMOSOME = "CHROMOSOME";
+    public static final String CDS = "CDS";
+
+    public static String formatExonStructure(GeneChromosomePosition chromosomePosition ){
+        if ( chromosomePosition.getOrientation() == '+')
+            return formatExonStructureForward(chromosomePosition);
+
+        return formatExonStructureReverse(chromosomePosition);
+
+    }
+
+    public static String printHTMLExonStructure(GeneChromosomePosition chromosomePosition ){
```

Comment:
Any documentation for this? What kind of html output does this produce? Perhaps we shouldn't have this specific output thing here in biojava but elsewhere